From: Rob Evers <revers@redhat.com>
Date: Mon, 4 Jan 2010 22:52:01 -0500
Subject: [scsi] lpfc: Update to version 8.2.0.61 driver release
Message-id: <20100104225208.24386.20167.sendpatchset@localhost.localdomain>
Patchwork-id: 22299
O-Subject: [RHEL5.5 PATCH 7/18] Update lpfc version for 8.2.0.61 driver release
Bugzilla: 549763
RH-Acked-by: Tomas Henzl <thenzl@redhat.com>
RH-Acked-by: Mike Christie <mchristi@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=549763

Update lpfc version for 8.2.0.61 driver release

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/lpfc/lpfc_version.h b/drivers/scsi/lpfc/lpfc_version.h
index d5aab43..07c1a76 100644
--- a/drivers/scsi/lpfc/lpfc_version.h
+++ b/drivers/scsi/lpfc/lpfc_version.h
@@ -18,7 +18,7 @@
  * included with this package.                                     *
  *******************************************************************/
 
-#define LPFC_DRIVER_VERSION "8.2.0.60"
+#define LPFC_DRIVER_VERSION "8.2.0.61"
 
 #define LPFC_DRIVER_NAME		"lpfc"
 #define LPFC_SP_DRIVER_HANDLER_NAME	"lpfc:sp"
